WebApp/src/components/StudyGroup/StudyGroup.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Toolbar from '../Toolbar/Toolbar';
+import Button from '@material-ui/core/Button';
+import NavigationIcon from '@material-ui/icons/Navigation';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Input from '@material-ui/core/Input';
+import './StudyGroup.scss';
+
+class StudyGroup extends React.Component {
+  state = {
+    sidebarOpen: true,
+    group: ''
+  }
+
+  Submit = () => {
+    this.props.history.push('/timetable');
+  }
+
+  handleSubmit = (evt) => {
+    this.setState({ group: evt.target.value });
+  }
+
+  canBeSubmitted = () => {
```

Comment:
You can rely here on JS coercion. Non-empty string is truthy, so `canBeSubmitted = () => this.state.group` would be fine. I suppose that its value needs to be boolean, because Button's "disable" prop accepts boolean. In that case you can convert to boolean like so: `canBeSubmitted = () => Boolean(this.state.group)` or `canBeSubmitted = () => !!this.state.group`